launcher/DesktopServices.cpp

```diff
@@ -84,4 +84,9 @@ bool isSnap()
 #endif
 }
 
+bool isGameScope()
+{
+    return qgetenv("XDG_CURRENT_DESKTOP") == "gamescope";
+}
```

Comment:
```suggestion
bool isSteamDeckGameScope()
{
    return qgetenv("XDG_CURRENT_DESKTOP") == "gamescope" && qgetenv("STEAM_DECK") == "1";
}
```

I think this better represents the intention here -- using fullscreen by default on the Steam Deck -- while also not making the very broad assumption that *all* gamescope sessions should be fullscreened

Reply:
Why does it need to be steam deck only?

Reply:
Well it could also be other handhelds, but we have no real way to check for that

The main point is that this shouldn't just be a blanket "gamescope == fullscreen" though, as people on a normal desktop may very well be using it, and they'd have no reason for Prism to be fullscreened. No other application does this to my knowledge

Reply:
Why wouldn't you want prism to render at native resolution on pc - but instead have blurry scaling?

Reply:
AFAIK the blurry scaling can be avoided depending on your gamescope configuration. There's also cases where Prism may not be the primary client of a gamescope session, which would make this behavior *super* annoying regardless

Maybe just add a proper toggle for this that defaults to true on the Deck?